src/app/board/board.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit, Input, Output, EventEmitter } from "@angular/core";
+import shuffle from "lodash/shuffle";
+
+@Component({
+  selector: "app-board",
+  templateUrl: "./board.component.html",
+  styleUrls: ["./board.component.css"]
+})
+export class BoardComponent implements OnInit {
+  @Input() numberOfTiles;
+  @Input() size;
+  @Output() boolToEmit = new EventEmitter<boolean>();
+
+  makeMenuVisible(bool) {
+    this.boolToEmit.emit(bool);
+  }
+
+  tilesSet: number[] = [...Array(this.numberOfTiles).keys()];
+
+  tiles: number[] = shuffle(this.tilesSet);
+
+  newGame() {
+    this.tiles = shuffle(this.tilesSet);
+  }
+
+  isZero(value) {
```

Comment:
You should use typing as much as possible 🙂 
```suggestion
  isZero(value: number) {
```